components/Ripple/src/MDCRippleTouchController.h

```diff
@@ -56,17 +56,45 @@
 @property(nonatomic, assign) BOOL shouldProcessRippleWithScrollViewGestures;
 
 /**
- Unavailable, please use `initWithView` instead.
+ Initializes the controller.
+
+ @param view The view that responds to the touch events for the ripple, and the view which the
```

Comment:
Please remove. There is no longer any `-init` method.

Reply:
Done